### PR TITLE
Allow executing ``npc_pilot_elite`` NPCs

### PR DESCRIPTION
**Pull request by NachosChipeados**
Allows players to execute pilot NPCs. Since every mod that uses ``npc_pilot_elite`` NPCs for something (see Gruntmode) , also adds similar changes to this, i think it'd be nice to bring this change into Northstar officially.

Before:
https://github.com/user-attachments/assets/489cc6f1-bc9f-4254-88a5-c30c7e30f6d6

After:
https://github.com/user-attachments/assets/b1263242-c6e8-403d-941c-8bed822ef5cc

Also added missing gib + headshot fx, to prevent weirdness.

Before:
https://github.com/user-attachments/assets/d8874ab2-fdeb-48eb-9db8-10526770f717

After:
https://github.com/user-attachments/assets/33eca4fd-3ec4-49a7-805a-024093efd068

HOW TO TEST:
1. Spawn a pilot NPC (``ent_create npc_pilot_elite``)
2. Try to execute them. Without this pr, you'll just punch them. With this pr, you'll properly execute them as expected.

Know issues:

* If firstperson executions are enabled, the game will create a duplicate pilot ragdoll after the animation's finished. See https://github.com/R2Northstar/NorthstarMods/issues/885
https://github.com/user-attachments/assets/68375bb5-29ed-4f59-b5fb-a5eb0a0fcc44

### CODE REVIEW

File: Northstar.Custom/keyvalues/scripts/aisettings/npc_pilot_elite.txt
Comment (GeckoEidechse) [inline Northstar.Custom/keyvalues/scripts/aisettings/npc_pilot_elite.txt:1]: Is that a modified file from a vanilla VPK or written completely from scratch?

**Review by Alystrasz — COMMENTED** [review]
I suspect the PR has side effects.

##### Without PR

[pr_main.webm](https://github.com/user-attachments/assets/46caf31d-4720-49c5-9c1d-67f19262dfb8)

##### With PR

[pr_with_main.webm](https://github.com/user-attachments/assets/f1cf0bb5-28e5-4696-b816-fdd2b5394040)

**Comment by NachosChipeados** [comment]
I haven't been able to recreate that in my end, so i'm not sure why it happens. Will investigate though

https://github.com/user-attachments/assets/3d0d984e-fc37-4e25-9627-3375d9cd6304

**Comment by NachosChipeados** [comment]
Yeah, i've got nothing. This pr doesn't modify their models or animations in any way, and i can't even repoduce the bug locally so i genuinely have no idea what's going on there

**Comment by NachosChipeados** [comment]
Update: while testing a completely unrelated pr, i managed to recreate the pilot elite bug, so i can confirm that it is unrelated to this pr and seems to be a bug with the vanilla game itself. I didn't even have this pr enabled while doing this, as you can see by the lack of exectuon prompt and headshot fx


https://github.com/user-attachments/assets/96de6821-7dbc-4de2-891e-42aabfa74e96

**Comment by Alystrasz** [comment]
I could reproduce the issue on another computer, with a fresh Titanfall2 + Northstar install.
Do you have any idea where this bug can come from?

**Comment by NachosChipeados** [comment]
Check #development in the discord

**Review by Alystrasz — APPROVED** [review]
Turns out behavior bugs are not introduced by this PR (see https://github.com/R2NorthstarTools/FlightCore/issues/1038 for more details).

I confirm this PR allows to execute `npc_pilot_elite` NPCs.

[npc_pilot_elite-execution.webm](https://github.com/user-attachments/assets/fac55dd9-e12b-436f-9139-0c101e732356)

**Comment by GeckoEidechse** [comment]
Merging based on reviews. Sorry it took so long ^^"

**Pull request merged**